js/main.js

```diff
@@ -53,14 +53,22 @@ window.search_data_loaded = function (search_data) {
 				resultsContent = "<p>No results found.</p>";
 				accessibleSummaryContent = "No results found."
 			}
+
 			$("#results-placeholder").html(resultsContent);
-			resultsElement.show();
-			contentElement.hide();
 			accessibleElement.html(accessibleSummaryContent);
+
+			if (resultsElement.is(":hidden")) {
+				// Display page title dynamically to avoid confusing bots that search
+				// for metadata
+				resultsElement.prepend("<h1>Search results</h1");
```

Comment:
You can start `resultsContent` with this heading.

Reply:
Nice one, done!